src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random randomizer = new Random();
+
+    public Figure getDefaultFigure() {
+        return (new Circle(Colors.WHITE,10));
+    }
+
+    public Figure getRandomFigure() {
+        int[] idx = new int[4];
```

Comment:
```suggestion
        int index = randomizer.nextInd(4);
```
ти можеш одразу задати межі рандому